code/modules/mob/living/carbon/human/species/human_species.dm

```diff
@@ -9,11 +9,10 @@
 	clothing_flags = HAS_UNDERWEAR | HAS_UNDERSHIRT | HAS_SOCKS
 	bodyflags = HAS_ICON_SKIN_TONE | HAS_BODY_MARKINGS
 	dietflags = DIET_OMNI
-	blurb = "Humanity originated in the Sol system, and over the last five centuries has spread \
-	colonies across a wide swathe of space. They hold a wide range of forms and creeds.<br/><br/> \
-	While the central Sol government maintains control of its far-flung people, powerful corporate \
-	interests, rampant cyber and bio-augmentation and secretive factions make life on most human \
-	worlds tumultous at best."
+	blurb = "Humanity, originating from the Sol system, has expanded over five centuries, establishing diverse colonies across vast space. \
+	These range from the militaristic Trans Solar Federation to the Union of Soviet Socialist Planets.<br/><br/> \
+	While the TSF predominantly governs human territories and is a founder of the Solar-Central Compact, corporate powers, widespread cyber and bio-augmentation, \
+	and clandestine groups render life beyond the TSF's core territories often tumultuous."
```

Comment:
This last sentence feels a little awkward, could you re-phrase it in some way?

Reply:
yeah re-reading it, its fucking garbo. I have reworked it to be alot less shit 

![C7Ta5S8uhy](https://github.com/ParadiseSS13/Paradise/assets/62493359/ea0bec7e-5caa-4be2-8008-3c50cb6f8832)

Reply:
ughghg i did another typo fix after posting this, but same shit p much